src/eclingo/parsing/parser.py

```diff
@@ -64,7 +65,12 @@ def __call__(self) -> None:
     def _parse_statement(self, statement: ast.AST) -> None:
         statement = self.theory_parser(statement)
         statement = parse_epistemic_literals_elements(statement)
-        statement = prefix_symbolic_atoms(statement, prefixes.U_PREFIX)
+        
+        if self.reification:
+            statement = reify_symbolic_atoms(statement, "u", reify_strong_negation=True)
```

Comment:
Let us use ```prefixes.U_PREFIX``` instead of "u", just in case we want to change it in the future.

Reply:
The problem with U_PREFIX is that is "u_", so if we use that it will add the underscore line. If we change it, then we will have to iterate for all the other tests that use "u_". 
I guess that at some point we will have, but I thought that maybe for now it was easier to just use "u" since the subset of tests that we are changing now is smaller than the overall.

Reply:
Let us define U_NAME as "u" and use it.
Also redefine U_PREFIX in terms of U_NAME.

Does this cause trouble?

Reply:
All good now